### DISCUSSION TITLE
I am facing some issue with fillament image upload and store it.

### DISCUSSION THREAD

**Discussion by mujahidhasanwc** [filamentphp/filament#7022] [General]
Here is I am trying to upload image in my local directory. It is saved but did not show the list.
I am trying this ways.
 FileUpload::make('image') it take image directly in public but the did not show in list when I am use it for show the image ImageColumn::make('image')->disk('public'), 
 After that I am trying with disk and directory the image save it in directory but the image name in database store with location.
 and the list did not show still .

**Comment by pboivin** [comment]
Here's a quick checklist for file uploads, can you see if you forgot any steps?
https://filamentphp.com/tricks/file-upload-previews-not-loading
